tests/set-labels/legacy-config/.expected/diff.patch

```diff
@@ -21,7 +21,4 @@ index 1e641f7..0dde4b3 100644
 +    fruit: apple
  spec:
    selector:
-     labels:
-       foo: bar
```

Comment:
this change is skeptical. Falling legacy function config to `SetLabels` should give the same result, right?

Reply:
nice fix!